backend/src/gee/caching_script.py

```diff
@@ -1,30 +1,99 @@
-import json
-
-content = [
-    {"timestamp": 1493251200, "value": 0.6265267304234295},
-    {"timestamp": 1494720000, "value": 0.68603163673333},
-    {"timestamp": 1494979200, "value": 0.755257128311451},
-]
-var_name = "msavi_daily_cache"
-file_name = "../cache/temp_cache.py"
-INDENT = "    "
-
-def combine_chache_with_update():
-    # update existing cache
-    return
-
-def write_year(file, year, results):
-    file.write(f"{INDENT}# Year {year}\n")
-
-def write_results_to_cache(results: list, var_name: str, file_name: str):
-    with open(file_name, "w") as file:
-        file.write(f"{var_name} = [\n")
-        for day in results:
-            file.write(f"{INDENT}{json.dumps(day)},\n")
+import os
+from enum import Enum
+
+from ..constants import IndexType
+from ..cache import ndvi_cache
+from ..cache import msavi_cache
+
+
+TIMESTAMP = "timestamp"
+CACHE_PATH = "../cache/"
+
+
+class ChosenCache(Enum):
+    NDVI = ndvi_cache.ndvi_daily_cache
+    MSAVI = msavi_cache.msavi_daily_cache
+
+
+index_type_to_cache = {
+    IndexType.NDVI: ChosenCache.NDVI,
+    IndexType.MSAVI: ChosenCache.MSAVI,
+}
+
+
+class CacheName(Enum):
+    NDVI = "ndvi_daily_cache"
+    MSAVI = "msavi_daily_cache"
+
+
+index_type_to_cache_name = {
+    IndexType.NDVI: CacheName.NDVI,
+    IndexType.MSAVI: CacheName.MSAVI,
+}
+
+
+class CacheFileName(Enum):
+    NDVI = "ndvi_cache.py"
+    MSAVI = "msavi_cache.py"
+
+
+index_type_to_cache_file = {
+    IndexType.NDVI: CacheFileName.NDVI,
+    IndexType.MSAVI: CacheFileName.MSAVI,
+}
```

Comment:
The enums are a good idea. For now, I thinks it is completly fine but I am wondering if `CacheFileName`, `CacheName`, `index_type_to_cache_file` and `index_type_to_cache_name` could be consolidated into one dictionary where the key is the chosen index and everything else can be found under this key.